rootfs/scheduler/__init__.py

```diff
@@ -770,8 +772,9 @@ def _create_service(self, name, app_name, app_type, data={}, **kwargs):
         image = kwargs.get('image')
         try:
             image = self.registry + '/' + image
+            repo = image.split(":")
             # image already includes the tag, so we split it out here
-            docker_cli.pull(image.rsplit(':')[0], image.rsplit(':')[1])
+            docker_cli.pull(repo[0]+":"+repo[1], tag=repo[2], insecure_registry=True)
```

Comment:
The internal registry is insecure and the docker engine demands that you explicitly set this flag when dealing with these, however I'm not sure why we need to set it in the CLI if the engine already has such information.